packages/keychain/src/components/ExecutionContainer.tsx

```diff
@@ -56,6 +58,8 @@ export function ExecutionContainer({
     "execute",
   );
 
+  const { isEditable, onToggleEditable } = useCreateSession();
```

Comment:
Imo it would be better to pass the right item element as a prop to execution container than to access `useCreateSession` here, so we don't couple session creation at this low level